jobs/config.json

```diff
@@ -9718,6 +9718,25 @@
       "sig-aws"
     ]
   },
+  "ci-kubernetes-e2e-kops-aws-encryption-provider": {
+    "args": [
+      "--cluster=e2e-kops-aws-encryption-provider.test-cncf-aws.k8s.io",
+      "--deployment=kops",
+      "--env=KUBE_SSH_USER=admin",
+      "--env=KOPS_FEATURE_FLAGS=+EnableKMSPlugin",
+      "--extract=ci/latest",
+      "--ginkgo-parallel",
+      "--kops-etcd-version=3.1.10",
```

Comment:
Do we need to specify the etcd version here?
--kops-etcd-version=3.1.10 is specified at ci-kubernetes-e2e-kops-aws-canary only and that comes from https://github.com/kubernetes/test-infra/commit/10753071a8a44f6e4315a72da0ac3091b3147b4c

Reply:
Yes, etcd v3 is a requirement for this test.